browser/layout/partials/ctm.html

```diff
@@ -2,7 +2,20 @@
 var classes = document.documentElement.className; {{!note: once we ditch support for ie9 we can just use classlist}}
 classes = classes.replace(/\bno-js\b/, 'js'); {{!note: needed for JS that's run out of the scope of the bootstrap (inline onclicks, etc)}}
 
-window.cutsTheMustard = (typeof Function.prototype.bind !== 'undefined' && typeof document.documentElement.dataset === 'object' && ('withCredentials' in new XMLHttpRequest()));
+var getRandomValuesAvailable;
+try {
+	getRandomValuesAvailable = 'getRandomValues' in window.crypto
+} catch (e) {
+	getRandomValuesAvailable = 'getRandomValues' in window.msCrypto
```

Comment:
Should this be updated in http://origami.ft.com/docs/developer-guide/modules/core-vs-enhanced-experience/#1-define-the-test ?